stl/inc/format

```diff
@@ -2355,14 +2348,12 @@ public:
 
 inline constexpr size_t _Fmt_buffer_size = 256;
 
-template <class _Iterator>
-struct _Back_insert_iterator_container_type {
-    using type = void;
-};
-
-template <class _Container>
-struct _Back_insert_iterator_container_type<back_insert_iterator<_Container>> {
-    using type = _Container;
+template <class _OutputIt>
+struct _Fmt_iterator_flush {
+    template <class _Ty>
+    static _OutputIt _Flush(const _Ty* const _First, const _Ty* const _Last, _OutputIt _Output) {
+        return _STD _Copy_unchecked(_First, _Last, _STD move(_Output));

```

Comment:
Why `_unchecked` here? Have we validated and unwrapped `_Output`? (I don't see how we *could* have, given that we are formatting piecemeal and haven't necessarily determined the entire length of the output.)

Reply:
If I recall it was already using copy unchecked, so I am not sure